### PR TITLE
Prepare Buildroot update to 2022.11

### PR DESCRIPTION
**Pull request by jforissier**
A couple of commits to prepare the update of Buildroot to the next version (2022.11) which is due soon. Tested with 2022.11-rc2.

This obviously depends on a manifest update, I will create a PR when 2022.11 is out.

### CODE REVIEW

File: br-ext/package/optee_examples_ext/optee_examples_ext.mk [hunk 1]
```diff
@@ -3,7 +3,7 @@ OPTEE_EXAMPLES_EXT_SOURCE = local
 OPTEE_EXAMPLES_EXT_SITE = $(BR2_PACKAGE_OPTEE_EXAMPLES_EXT_SITE)
 OPTEE_EXAMPLES_EXT_SITE_METHOD = local
 OPTEE_EXAMPLES_EXT_INSTALL_STAGING = YES
-OPTEE_EXAMPLES_EXT_DEPENDENCIES = optee_client_ext host-python3-pycryptodomex host-python-cryptography
+OPTEE_EXAMPLES_EXT_DEPENDENCIES = optee_client_ext host-python-cryptography
```
Comment (jbech-linaro) [inline br-ext/package/optee_examples_ext/optee_examples_ext.mk:6]: Here we go again 😉

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by jforissier** [comment]
@etienne-lms let me know if you're OK with what I consider is a minor inconvenience, mentioned in the description of "buildroot: remove opensc external package". Or if you have an idea to address that particular issue. Thanks!

**Review by etienne-lms — COMMENTED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by etienne-lms** [comment]
> @etienne-lms let me know if you're OK with what I consider is a minor inconvenience, mentioned in the description of "buildroot: remove opensc external package". Or if you have an idea to address that particular issue. Thanks!

It was for convenience when using the tool but it's not something that affects our pkcs11 support or CI tests (xtest), so ok for me.

**Pull request merged**